p3/migrations/0001_initial.py

```diff
@@ -0,0 +1,92 @@
+                ('amount', models.DecimalField(decimal_places=2, max_digits=6)),
+                ('message', models.TextField(blank=True)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='assopy.User')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='P3Profile',
+            fields=[
+                ('profile', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, related_name='p3_profile', serialize=False, to='conference.AttendeeProfile')),
+                ('tagline', models.CharField(blank=True, help_text='describe yourself in one line!', max_length=60)),
+                ('twitter', models.CharField(blank=True, max_length=80)),
+                ('image_gravatar', models.BooleanField(default=False)),
+                ('image_url', models.URLField(max_length=500)),
+                ('country', models.CharField(blank=True, db_index=True, default=b'', max_length=2)),
+                ('spam_recruiting', models.BooleanField(default=False)),
+                ('spam_user_message', models.BooleanField(default=False)),
+                ('spam_sms', models.BooleanField(default=False)),
+                ('interests', taggit.managers.TaggableManager(help_text='A comma-separated list of tags.', through='conference.ConferenceTaggedItem', to='conference.ConferenceTag', verbose_name='Tags')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='P3Talk',
+            fields=[
+                ('talk', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, related_name='p3_talk', serialize=False, to='conference.Talk')),
+                ('sub_community', models.CharField(choices=[(b'', b'All'), (b'pydata', b'PyData')], default=b'', max_length=20)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SpeakerConference',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_time', models.BooleanField(default=False)),
+                ('speaker', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='p3_speaker', to='conference.Speaker')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Sprint',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=150)),
+                ('abstract', models.TextField(blank=True)),
+                ('conference', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='conference.Conference')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='assopy.User')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SprintPresence',
```

Comment:
same here: TODO: AFAIK this model is not used, we can probably remove it altogether.